src/lib/pickles/pickles.ml

```diff
@@ -272,36 +272,13 @@ module Prover = struct
     -> 'proof
 end
 
-module Proof_system = struct
-  type ( 'a_var
-       , 'a_value
-       , 'max_proofs_verified
-       , 'branches
-       , 'prev_valuess
-       , 'widthss
-       , 'heightss )
-       t =
-    | T :
-        ('a_var, 'a_value, 'max_proofs_verified, 'branches) Tag.t
-        * (module Proof_intf with type t = 'proof and type statement = 'a_value)
-        * ( 'prev_valuess
-          , 'widthss
-          , 'heightss
-          , 'a_value
-          , 'proof )
-          H3_2.T(Prover).t
-        -> ( 'a_var
-           , 'a_value
-           , 'max_proofs_verified
-           , 'branches
-           , 'prev_valuess
-           , 'widthss
-           , 'heightss )
-           t
-end
-
-module Make (A : Statement_var_intf) (A_value : Statement_value_intf) = struct
-  module IR = Inductive_rule.T (A) (A_value)
+module Make
+    (A : Statement_var_intf)
+    (A_value : Statement_value_intf)
```

Comment:
Rename these to `Statement_var` and `Statement_value`? or `Arg_var`/`Arg_value` to go with the function analogy. Makes it slightly easier to read the code below